diff --git a/apis/charts_graphs_google_charts/charts_graphs_google_charts.class.inc b/apis/charts_graphs_google_charts/charts_graphs_google_charts.class.inc
index 76887c9..db819cb 100644
--- a/apis/charts_graphs_google_charts/charts_graphs_google_charts.class.inc
+++ b/apis/charts_graphs_google_charts/charts_graphs_google_charts.class.inc
@@ -233,7 +233,7 @@ class ChartsGraphsGoogleCharts extends ChartsGraphsCanvas {
       $this->parameters_to_send[CHARTS_GRAPHS_GOOGLE_CHARTS_MARKER_STYLE] = implode('|', $fill_colour);
     }
 
-    if ($this->legend_pos) {
+    if (isset($this->legend_pos)) {
       switch ($this->legend_pos) {
         case 'top':
           $chdlp = 't';
@@ -255,7 +255,7 @@ class ChartsGraphsGoogleCharts extends ChartsGraphsCanvas {
       }
       $this->parameters_to_send[CHARTS_GRAPHS_GOOGLE_CHARTS_LEGEND_POSITION] = $chdlp;
     }
-    if ($this->title_colour) {
+    if (isset($this->title_colour)) {
       $this->parameters_to_send[CHARTS_GRAPHS_GOOGLE_CHARTS_CHART_TITLE_COLOUR] = $this->title_colour;
     }
   }
